nginx/sites-enabled/hashbang-http

```diff
@@ -0,0 +1,18 @@
+# Hashbang server configuration
+# ref: https://gist.github.com/RyanSquared/e59c7e274abef06e63b0e47e46997b00
+#
+server {
+	listen 80;
+	listen [::]:80;
+
+	server_name "~^(?<user>[a-zA-Z0-9]+)\.(?<server>\w{2}\d{1})\.hashbang\.sh$";
+
+	root /home/$user/Public;
```

Comment:
Drop this, as discussed this is either non-working (Nginx doesn't have access to `/home/$user`) or dangerous.